__docs__/wonder-blocks-cell/compact-cell.stories.tsx

```diff
@@ -317,6 +317,73 @@ CompactCellDisabled.parameters = {
     },
 };
 
+/**
+ * These are `CompactCell` instances with custom background colors. Note that we
+ * use the `style` prop to pass a custom style object to the cell.
+ *
+ * We recommend using a faded background color (third cell) to make the cell
+ * look as expected with different states (e.g. hover, focus, active).
+ *
+ * If you use a solid background color (last cell), the cell states will not
+ * change the background color.
+ */
+export const CompactCellsAsListItems: StoryComponentType = {
+    render: () => (
+        <section role="list">
+            <View role="listitem">
```

Comment:
**question**: Out of curiosity, why did you use `section` and `View` with `role="list"` and `role="listitem"` respectively, instead of just using `ul` and `li`?

Reply:
hmmm great question... I was replicating the code where this was originally introduced in `webapp` so I could make sure that the issue is fixed, but I didn't think very carefully about how the code was created. The `View` part is to get View's default styles, which make sense for the `listitem` case, but the `section` is incorrectly used, and could probably use something different.

Side note: I've noticed that sometimes we tend to overuse `View` instead of relying on semantic elements via the `addStyle` function. This could have used `ul/li` in that case.

Reply:
I've updated the docs to reflect this and recommend this `addStyle` approach.